internal/api/utils.go

```diff
@@ -33,3 +37,9 @@ func StringPtrSliceToStringSlice(s []*string) []string {
 	}
 	return out
 }
+
+// Check that a string conforms to the UUID format
+func IsValid(u string) bool {
+	err := uuid.Validate(u)
+	return err == nil
+}
```

Comment:
Suggest either eliminating this function and just doing `if uuid.Validate(u) != nil` where needed, or else rename the function to `UUIDIsValid` to clarify _what_ is valid.